mongo/change_stream.go

```diff
@@ -451,7 +451,7 @@ func (cs *changeStream) Decode(out interface{}) error {
 		return err
 	}
 
-	return bson.UnmarshalWithRegistry(cs.coll.registry, br, out)
```

Comment:
In the change stream cursor object, the collection field `coll` is `nil`, when we open a change stream watch on any database or on any client. 

And hence, this will pop up a panic error.

Reply:
```suggestion
	return bson.UnmarshalWithRegistry(cs.db.registry, br, out)
```
Therefore, using database registry will solve the panic problem.